docs/understanding-tracking-design/versioning-your-data-structures/enterprise/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Versioning Data Structures in BDP Enterprise"
```

Comment:
Do we really need two different pages? Can this be a single page with two sections, or with tabs? I imagine that in near future we might bring some of the Cloud functionality into Enterprise, and then two separate pages will no longer make sense.

Reply:
We're trying to hide the complexity of versioning Json for Cloud users atm. And besides the current docs are not great, they don't cover all changes. So I'd prefer to leave it separate for the time being and return to it later on.